src/pages/blog/stripe-docs/index.mdx

```diff
@@ -0,0 +1,121 @@
+
+Stripe’s documentation—and the impact it has had on the company’s user experience, retention, and brand attributes to why the company's success.
+
+But Stripe isn’t alone in its goal of building the best docs. Many companies aspire to do the same but fail somewhere along the way. Why?
+
+I went down this research rabbit hole, and here’s how Stripe created the culture and habits that enabled their team of engineers to write beautiful docs.
+
+## Foster a strong writing culture
+
+There are few companies that have embedded writing into their culture more than Stripe. The company has [a literal press](https://press.stripe.com/) that publishes books, [an engineering magazine](https://increment.com/), and one of the most widely read [engineering blogs](https://stripe.com/blog) of any company in Silicon Valley.
+
+<img
+  src="/static/images/blog/stripe-docs/twitter.png"
+  alt="blog thumbnail"
+  className="w-full rounded-md"
+/>
+
+As former Stripe employee [Patrick McKenzie](https://www.kalzumeus.com/2019/3/18/two-years-at-stripe/), put it: “Stripe is a celebration of the written word which happens to be incorporated in the state of Delaware.”
+
+This culture starts at the top, according to Dave Nunez, Stripe’s former Documentation Manager. “The first emails I saw from our CEO [Patrick Collison] literally had footnotes,” he told [Slab](https://slab.com/blog/stripe-writing-culture/) in an interview. “He structured his emails to be like research papers and put the peripheral information at the bottom so as not to detract from the core information.”
+
+For companies that want to build better docs, Nunez believes culture change is the biggest lever available. “I wouldn’t recommend early-stage startups just go out and hire a full-time docs writer. I’ve learned that, especially in high-performance environments, changing the culture is the highest leverage investment you can make,” he told [First Round Review](https://review.firstround.com/investing-in-internal-documentation-a-brick-by-brick-guide-for-startups).
+
+It’s worth noting that this takes work—and in some cases, behavior change—from leaders in a company.
+
+“I’ve seen executives write emails all lowercase, loaded with abbreviations and typos, which gives the impression that the time they saved by writing a low-quality email is more valuable than the time it’s going to take others to decipher it. Not only are people frustrated with the poor readability, but other employees will learn not to take writing seriously either,” Nunez told First Round Review.
+
+**My takeaway**: When building a culture, actions speak louder than words. In order to get people to prioritize writing quality you have to demonstrate thoughtfulness and attention to detail every day. This is especially true for leaders who set the tone for everyone else.
```

Comment:
Somewhere at this point in the article...

Thoughts on incorporating some hand wave-y stats from us into the stripe article as well?
I’m thinking it would be sick if in the section where we’re talking about stripe’s writing culture if we can add a stat like 90% of calls we have with companies who want better docs includes someone from the C-Suite.